webapp/lib/main_ui.dart

```diff
@@ -93,8 +102,7 @@ class CodaUI {
       displayDatasetHeadersView(dataset);
     } catch (e, s) {
       displayUrlErrorView(e.toString());
-      log.verbose(e.toString());
-      log.verbose(s.toString());
+      log.severe('$e\n$s');
```

Comment:
as above, please could we JSON encode this?

Reply:
Done, thanks